### ISSUE TITLE
Awesome TLS error

### ISSUE THREAD

**Issue by 2xiaodi** [sleeyax/burp-awesome-tls#72]
Awesome TLS error: dial tcp 218.98.12.220:80: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. 

![image](https://github.com/sleeyax/burp-awesome-tls/assets/57615079/0f52ce02-d73d-49f0-b49a-da38de053390)
When I visit this site through the plugin will return this error, please help to see what is the problem

**Comment by sleeyax** [comment]
**TL;DR** The solution is to connect with a Japanese VPN or proxy enabled to bypass geo-restricitons imposed by this site. Awesome TLS, default burp or any regular browser can't connect to this site otherwise. 

---

This site doesn't have HTTPS enabled which may cause issues with Awesome TLS:

![image](https://github.com/sleeyax/burp-awesome-tls/assets/30344294/55eb373f-73dc-411e-9011-46f8dfacad84)

Furthermore, loading the site with Awesome TLS disabled gets me another error:

![image](https://github.com/sleeyax/burp-awesome-tls/assets/30344294/0fcfac4c-e3d7-4952-8ae7-f127ab562254)

And when loading the address in a regular browser it can't seem to resolve the address and/or connect to it.

---

That being said, this site has some issues and is clearly leveraging geo-blocking to mandate access. Try again with a system-wide VPN or proxy to a Japanese server. It worked on my end:

![image](https://github.com/sleeyax/burp-awesome-tls/assets/30344294/0b8fe4f0-3e6c-4441-b9d8-d19c00415820)

**Comment by 2xiaodi** [comment]
Thank you very much. I can also access the above site with a Japanese agent, http://9527yyds.com:81/, but this is the same framework but the agent cannot access it. By the way, how did you find out that the Japanese agent can access it and what tools do you rely on? I'd appreciate it if you could share it 
![image](https://github.com/sleeyax/burp-awesome-tls/assets/57615079/877f7dbd-e233-4a9f-bd06-04314b869f42)

**Comment by sleeyax** [comment]
@2xiaodi I believe you commented something because I got an email notification. But did you removed your comment afterwards? 

Anyways, if you have any more questions just ask. Otherwise this issue can be closed.
